apps/bot/db/actions/messages.ts

```diff
@@ -76,7 +76,7 @@ export const deleteMessage = async (messageId: string, postId: string) => {
 }
 
 export const markMessageAsSolution = async (
-  messageId: string,
+  messageId: string | false,
```

Comment:
Instead of false we could use `null`, I think it makes more sense as we are basically saying that the new answer is nothing

Reply:
Yeah I realised that it was a little confusing...